WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -195,6 +196,19 @@ final class SiteInformationWizardContent: UIViewController {
     private func goNext() {
         let collectedData = SiteInformation(title: titleString(), tagLine: taglineString())
         completion(collectedData)
+        trackBasicInformationNextStep()
+    }
+
+    private func trackBasicInformationNextStep() {
+        if nextStep.isPrimary {
+            var basicInformationProperties = [String: AnyObject]()
```

Comment:
The same comment I made in `ErrorStateViewController.swift` applies for this dictionary.